### ISSUE TITLE
UnicodeDecodeError: 'utf8' codec can't decode ... mopidy/m3u/playlists.py

### ISSUE THREAD

**Issue by r0k** [mopidy/mopidy#1209]
After apt-get updating my debian jessie and updating all pip's on a beaglebone black recently mopidy didnt started up anymore. The console showed an exception like __UnicodeDecodeError: 'utf8' codec can't decode ... mopidy/m3u/playlists.py__ during loading the modules configured in mopidy.conf

Research showed that my local stored playlist filename contained a "special char" _(the german "ö")_ in a wrong coding which caused mopidy startup to fail (filename, not file content).

The playlist has been written with the previous configuration, worked fine, and was broken after the last update.

Best Regards,
Oliver

**Comment by jodal** [comment]
This is probably enough info to go on to fix this, but do you have the complete stack trace too?

**Comment by jodal** [comment]
What version of Mopidy are you running? #1175 is a similar error fixed in Mopidy 1.0.5.

**Comment by r0k** [comment]
Jodal, 

sorry for the delay. Its 1.0.7 i am running and the bug #1175 seems to cover unicode titles inside a playlist. My described bug is caused by a "wrong" character / character encoding in the file name of the playlist itself.

The playlist has been written previously by mopidy (mopidy-mobile frontend) and crashed on startup  after updating. __UnicodeDecodeError: 'utf8' codec can't decode__ ... __mopidy/m3u/playlists.py__
Renaming the playlist file fixed the bug and made mopidy to come up again. 

I have tried to reproduce the bug by renaming a playlist file in several ways to force the mentioned exception and a stack trace... nothing worked. It seems when using the command line the file / filename encoding is always right. Tried: ö, Ã¶, �.
